### DISCUSSION TITLE
Revision of the lists in Mastodon(?)

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#18578] [General]
While trying out some Mastodon functions, I noticed the list function in Mastodon and it occurred to me that this could use some revision.

Could the list function in Mastodon perhaps be adapted so that, for example, hashtags can be added ($user searches for hashtag and would like to save posts of this hashtag in the future in a list that he can select so that he wants to stay up to date with this hashtag, but does not want to look at his timeline (TL) for this?

**Comment by StampedingLonghorn** [comment]
Bump. I would like this feature as well.
